src/components/atoms/FormModal/FormModal.tsx

```diff
@@ -26,9 +30,9 @@ interface FormModalProps extends FormProps {
 
 const FormModal = ({
   title,
-  visible,
   className,
   width,
+  visible,
```

Comment:
there is no longer a `visible` prop, so you could use `useUnmount` right?

Reply:
I don't get it, where should I use the `useUnmount`

Reply:
Right now we have this code:
```
useEffect(() => {
    if (!visible) {
      setIsSubmitting(false);
    }
  }, [setIsSubmitting, visible]);
```
but as you don't have a visible prop anymore this code will not work. So you could change this to just happen on unmount

Reply:
But I re-added the visible prop now to use it in the HOC. So I updated the `useEffect`. I think it should be fine. right?

Reply:
but the visible prop is not passed into the FormModal, try to console.log it.

Reply:
Yes true, My bad. I have added the `useUnmount`